src/librehatti/reports/register.py

```diff
@@ -116,7 +116,7 @@ def daily_report_result(request):
             request_status = request_notify()
             start_date = datetime.strptime(start_date, '%Y-%m-%d').strftime('%B-%d-%Y')
             end_date = datetime.strptime(end_date, '%Y-%m-%d').strftime('%B-%d-%Y')
-            back_link=reverse('librehatti.reports.register.daily_report_result')
+            back_link=reverse('daily_report_result')
```

Comment:
Why is here different syntax than your other changes?

For example:
        reverse("librehatti.**catalog**.views.**nonpaymentordersuccess**"))
        reverse("**catalog**:**nonpaymentordersuccess**"))

I see that those use "views" and this one uses "reports.register".

Reply:
> Why is here different syntax than your other changes?

Because these urls are directly given in main urls.py of app 'librehatti'. So, these can be directly called without namespacing.

> For example:
> reverse("librehatti.**catalog**.views.**nonpaymentordersuccess**"))
> reverse("**catalog**:**nonpaymentordersuccess**"))
> 
> I see that those use "views" and this one uses "reports.register".
> 
> Is there any other option to specify it? Won't it conflict with `daily_report_result` from any other app?

No, because to call it for other app, we need to use namespacing.